### PR TITLE
Vendor javascript dependencies.

### PR DESCRIPTION
**Pull request by stuartnelson3**
requested in #305

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -12,6 +12,9 @@
 // (https://github.com/sstephenson/sprockets#sprockets-directives) for details
 // about supported directives.
 //
+//= require jquery.min
+//= require jquery-ui.min
+//= require angular.min
```
Comment (juliusv) [inline app/assets/javascripts/application.js]: Since there is no Gemfile update, where do these come from?

Comment (stuartnelson3) [inline app/assets/javascripts/application.js]: at the bottom of this "files" list, they were added to vendor/assets/javascripts.

Comment (juliusv) [inline app/assets/javascripts/application.js]: Ah, of course. See, me and my Rails skills.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**